web/packages/shared/components/DesktopSession/Withholder.tsx

```diff
@@ -42,22 +42,23 @@ export class Withholder {
    * The internal array of keystrokes that are currently
    * being withheld.
    */
-  private withheldKeys: Array<WithheldKeyboardEventHandler> = [];
+  private withheldKeys: Array<WithheldInputEventHandler> = [];
 
   /**
-   * All keyboard events should be handled via this function.
+   * All input events should be handled via this function.
    */
-  public handleKeyboardEvent(
-    params: KeyboardEventParams,
-    handleKeyboardEvent: (params: KeyboardEventParams) => void
+  public handleInputEvent(
+    params: InputEventParams,
+    handleInputEvent: (params: InputEventParams) => void
   ) {
-    const key = params.e.key;
-
-    // If this is not a key we withhold, immediately flush any withheld keys
+    // If this is not a key we withhold or a mouse event, immediately flush any withheld keys
     // and handle this key.
-    if (!this.keysToWithhold.includes(key)) {
+    if (
+      params.e instanceof MouseEvent ||
+      !this.keysToWithhold.includes((params.e as KeyboardEvent).key)
```

Comment:
TS narrows down the type:
```suggestion
      !this.keysToWithhold.includes(params.e.key)
```